web/console/src/app/components/common/Registration/AuthWrapper/index.tsx

```diff
@@ -3,7 +3,6 @@
 
 import { useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
-import { toast } from 'react-toastify';
 
 // @ts-ignore
 import KeyStorageHandler from '../../../../velas/keyStorageHandler';
```

Comment:
1) Do we need ts-ignore? 
2) Allias for route

Reply:
1) yes, we need, because these files are provided by velas and they are js files, not ts
2) i will fix it in pr with minting service